### ISSUE TITLE
Trying to use with TS, but get this error

### ISSUE THREAD

**Issue by Wiklem** [sarkistlt/mongoose-schema-to-graphql#26]
error TS2345: Argument of type '{ name: string; description: string; class: string; schema: any; }' is not assignable to parameter of type 'ConfigurationObject'.
  Types of property 'class' are incompatible.
    Type 'string' is not assignable to type 'ClassFieldType'.

**Comment by rpCal** [comment]
Same here. I've fix that by change in index.d.ts (JUST QUICK FIX)

```js
type ConfigurationObject = {
        name: string,
        description?: string,
        class: ClassFieldType,
        schema: mongoose.Schema,
        exclude?: RegExp | string[],
        extend?: ExtendFieldType,
        fields?: ObjectKeyStringValueAnyType,
    };

```

```js
    type ConfigurationObject = {
        name: string,
        description?: string,
        class: string,
        schema: mongoose.Schema,
        exclude?: RegExp | string[],
        extend?: ExtendFieldType,
        fields?: ObjectKeyStringValueAnyType,
    };
```

**Comment by sarkistlt** [comment]
should be fixed in v2.8.1

**Comment by EnOane** [comment]
You don't resolve this problem
